src/library_fs.js

```diff
@@ -1715,7 +1715,7 @@ FS.staticInit();` +
           if (datalength !== chunkSize) xhr.setRequestHeader("Range", "bytes=" + from + "-" + to);
 
           // Some hints to the browser that we want binary data.
-          if (typeof Uint8Array != 'undefined') xhr.responseType = 'arraybuffer';
+          xhr.responseType = 'arraybuffer';
```

Comment:
This is unrelated by I notices we were trying to handle older browsers where Uint8Array did not exist.. I'm pretty sure we don't support that anymore, right?

Reply:
Yes, even IE11 has typed arrays. We don't even try to support things older than that. There is no efficient way to polyfill typed arrays using normal arrays in any case...